certbot/tests/auth_handler_test.py

```diff
@@ -254,8 +254,8 @@ def mock_now_effect():
 
         assert mock_sleep.call_count == 3 # 1s, 20m and 10m sleep
         assert mock_sleep.call_args_list[0][0][0] == 1
-        assert abs(mock_sleep.call_args_list[1][0][0]-interval - 1) < 1
-        assert abs(mock_sleep.call_args_list[2][0][0]-interval/2 - 1) < 1
+        assert abs(mock_sleep.call_args_list[1][0][0] - (interval - 1)) < 1
+        assert abs(mock_sleep.call_args_list[2][0][0] - (interval/2 - 1)) < 1
 
```

Comment:
Mathematically, aren't these lines (before and after) exactly the same?

Reply:
Division does go before subtraction, but that wasn't the problem. It's the final -1.
```
- (interval - 1) = -interval + 1
```